recipes/drflac/all/conanfile.py

```diff
@@ -27,21 +31,11 @@ class DrflacConan(ConanFile):
         "no_crc": False,
         "no_ogg": False,
         "no_simd": False,
-        "no_stdio": False
+        "no_stdio": False,
+        "no_wchar": False
     }
-    generators = "cmake"
     exports_sources = ["CMakeLists.txt", "dr_flac.c"]
```

Comment:
```suggestion
    exports_sources = ["CMakeLists.txt"]
```
downloaded files don't need to be exported